pages/c/_cluster/istio/index.vue

```diff
@@ -80,12 +91,39 @@ export default {
               :rel="rel"
               @click.stop
             >
-              <t k="istio.links.label" />
+              <t k="istio.links.kiali.label" />
+              <i class="icon icon-external-link pull-right" />
+            </a>
+            <hr />
+            <div class="description">
+              <span v-html="t('istio.links.kiali.description', {link: monitoringUrl}, true)" />
+            </div>
+          </div>
+        </span>
+      </div>
+      <div :class="{'disabled':!jaegerUrl}" class="box link-container">
+        <span
+          class="link-content pull-right"
+          @click="$refs.jaeger.click()"
+        >
+          <div class="link-logo">
+            <img :src="jaegerLogo" />
```

Comment:
This is fine but FYI if the src isn't dynamic during run-time, you can directly put in `<img src="~/assets/images/..." />` and it gets resolved at build-time.